public/app/features/trails/DataTrailsHistory.tsx

```diff
@@ -148,15 +151,24 @@ export class DataTrailHistory extends SceneObjectBase<DataTrailsHistoryState> {
             return;
           }
 
-          this.addTrailStep(
-            trail,
-            'time',
-            parseTimeTooltip({
-              from: newState.from,
-              to: newState.to,
-              timeZone: newState.timeZone,
-            })
-          );
+          const tooltip = parseTimeTooltip({
+            from: newState.from,
+            to: newState.to,
+            timeZone: newState.timeZone,
+          });
+
+          this.addTrailStep(trail, 'time', tooltip);
+
+          if (config.featureToggles.unifiedHistory) {
+            appEvents.publish(
+              new HistoryChangedEvent({
```

Comment:
I find this naming a bit confusing name. History has not really changed, I would find it more clear if it was named something like "CreateHistoryStepEvent" or "RecordHistoryStepEvent" 

HistoryChangedEvent I would assume mutates an existing history in some way

Reply:
Hi @torkelo,
Makes sense to me. I changed it to `RecordHistoryEntryEvent` since the interface is called `HistoryEntryView`, so we keep them similar/related.